Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,293 @@
+description: Minerva eliminates the endpoint security gap while empowering companies
+  to embrace technology fearlessly.
+detaileddescription: |-
+  Minerva Labs integration with Demisto.
+  The integration queries the Minerva server for new events, endpoint status, groups and able to create active response such as adding vaccinations and exclusions.
+configuration:
+- display: url
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: user
+  name: user
```

Comment:
Please use the `Authentication` parameter type for credentials and make it mandatory.

Reply:
Couldn't find relevant documentation / demo regarding it, but by some guess work it seems to work.
Hope it's ok now.

Reply:
@danikdanik you've raised a good point though, so we've created the following:
https://github.com/demisto/content/blob/master/docs/parameter_types/README.md